apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -876,12 +912,14 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
         return wasAcceptedBeforeLastFlip != _isAccepted(_vote, setting);
     }
 
+   // Note: maybe we should rename these "within" functions as they are not really "within" (they only check a single bound, rather than two)
```

Comment:
Minor gripe about the naming that may get solved with the above "refactor"; these functions act more like "have we passed this time checkpoint yet" rather than "am I within this time bound"?

Reply:
Addressed in 9609ef02a9a73dab443906adfe0f2b383a2e2acf